### ISSUE TITLE
Definition of virtual keyboard: Rework the Note for Types of Input

### ISSUE THREAD

**Issue by ljoakley** [w3c/wcag2ict#397]
The sentence is backwards, the user is presented with a list of input modes with the explanation of why they are listed at the end of the sentence. The user has to get to the end of the sentance to understand why these modes are listed. I would change the sentence as follows:  All of the following, and not limited to,have all been used by virtual keyboards as input that generates "keystroke" output: Speech, eye-gaze, sip-and-puff, sounds, switches, and codes.

**Comment by bruce-usab** [comment]
@ljoakley I changed the note to an example and reworded as follows:

> Speech, eye-gaze, sip-and-puff (and other kinds of switches), sounds, and morse code have all been used as ways to generate keystroke output.

I agree it read backwards, but I would like to fix that while keeping the examples at the front of the sentence.

**Comment by GreggVan** [comment]
Hmmm
we decided to write I  plain language so we need to practice this.   But it is nice to get to the meat sooner without as sentence like   "Throw the cow over the fence some hay"  where you need to get the end and then re-read the sentence (visually or in your head).

how about maybe

 Some of the many ways to generate keystroke input include speech, eye-gaze, sip-and-puff (and other kinds of switches), sounds, morse code, and, of course, keyboards (small, large, physical, on-screen, floating in the air, etc),

**Comment by bruce-usab** [comment]
This issue was [discussed on call](https://www.w3.org/2024/07/18-wcag2ict-minutes.html#t06).
- I like Gregg's rewrite better than my own.
- @samogami prefers current phrasing.

@maryjom plans to survey soon.
1. Is an edit warranted?
2. If so, what should the change be?

**Comment by bruce-usab** [comment]
This issue was [discussed on today's call](https://www.w3.org/2024/08/15-wcag2ict-minutes.html#t07).  I volunteered to put more details into this GitHub thread.

**Comment by bruce-usab** [comment]
Thank you @ljoakley for your review and comment.  We have decided not to update the definition of “virtual keyboard”.

The existing note makes a clear distinction between input and output.  Trying to remove the passive grammar we could not find a way to maintain that clear distinction.

Also, our attempts at rewrites were not concise.  One semantic difficulty is that, while useful, the list is _not_ examples of the defined term.  We came to consensus on not changing the text, that the text is clear enough as-is.
